tkn pac resolve -f fail on GetConfigFromConfigMap

![image](https://user-images.githubusercontent.com/98980/148978189-2ca198d7-23ea-46a7-bbd0-9166c99905e5.png)
```
% tkn pac resolve -f ./run.yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa8 pc=0x13dd706]

goroutine 1 [running]:
github.com/openshift-pipelines/pipelines-as-code/pkg/params.(*Run).GetConfigFromConfigMap(0xc00031fc70, {0x1a9d690, 0xc000040128})
	/home/chmouel/GIT/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/params/run.go:31 +0x46
github.com/openshift-pipelines/pipelines-as-code/pkg/cmd/tknpac/resolve.Command.func1(0xc000429b80, {0x1871c6f, 0x2, 0x2})
	/home/chmouel/GIT/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/cmd/tknpac/resolve/resolve.go:75 +0x9f
github.com/spf13/cobra.(*Command).execute(0xc000429b80, {0xc0003e7520, 0x2, 0x2})
	/home/chmouel/GIT/go/src/github.com/openshift-pipelines/pipelines-as-code/vendor/github.com/spf13/cobra/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc000428c80)
	/home/chmouel/GIT/go/src/github.com/openshift-pipelines/pipelines-as-code/vendor/github.com/spf13/cobra/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/chmouel/GIT/go/src/github.com/openshift-pipelines/pipelines-as-code/vendor/github.com/spf13/cobra/command.go:902
main.main()
	/home/chmouel/GIT/go/src/github.com/openshift-pipelines/pipelines-as-code/cmd/tkn-pac/main.go:14 +0xcb
```

--- a/pkg/cmd/tknpac/resolve/resolve.go
+++ b/pkg/cmd/tknpac/resolve/resolve.go
@@ -70,9 +70,7 @@ func Command(run *params.Run) *cobra.Command {
 				if !noConfigErr {
 					return err
 				}
-			}
-
-			if err := run.GetConfigFromConfigMap(ctx); err != nil {
+			} else if err := run.GetConfigFromConfigMap(ctx); err != nil {
 				log.Printf("Warning: cannot get pipelines-as-code config map in pipelines-as-code namespace. Using defaults. Error: %v\n", err)
 			}
 

